taskcat/_cli_modules/test.py

```diff
@@ -118,7 +125,11 @@ def run(
             for test in buckets.values():
                 for bucket in test.values():
                     if bucket.name not in deleted:
-                        bucket.delete(delete_objects=True)
+                        _bucket_delete_evaluation(
+                            config.config.project.s3_regional_buckets,
+                            bucket.empty,
```

Comment:
Emptying the shared buckets will break concurrent taskcat runs. Either just leave the objects behind, or only delete objects created in this run. I'm partial to having them left behind, so then I can manually launch a stack off the same s3 objects post-test.

Reply:
I'll opt for leaving them behind; Determining the objects added this run could be a little complicated without more logic. 

Something I think we want to look at moving forward, but for now.. this will work.